include/ultra64/gbi.h

```diff
@@ -10,7 +10,9 @@
 /* Private macro to wrap other macros in do {...} while (0) */
 #define _DW(macro) do {macro} while (0)
 
-#define F3DEX_GBI_2
+#ifndef F3DEX_GBI
+ #define F3DEX_GBI_2
+#endif
 
 #ifdef    F3DEX_GBI_2
 # ifndef  F3DEX_GBI
```

Comment:
you can remove this part defining `F3DEX_GBI` by default since you added `F3DEX_GBI_2` defined by default

Reply:
^

Reply:
I cant get it to even build without this added here. If anyone else has a solution that is cleaner I would be willing to use it instead.

Reply:
my bad, I always assumed it was one or the other but it's actually "incremental" I guess, like `F3DEX_GBI_2` implies `F3DEX_GBI`

Reply:
I mean there probably is a better way. I just dont know what it is.